meta/htmlMinifier.js

```diff
@@ -210,6 +212,141 @@ function getTrimmedAttributes(element) {
 	return trimmedAttributes;
 }
 
+/**
+ * @typedef {Object} PrimitiveTypes
+ * @property {Array<string>} supportedPrimitives - An array of supported primitive actions for the element.
+ */
+
+/** @typedef {Object} PrimitivesAttribute
+ * @property string [data-supported-attributes]
+ */
+
+/**
+ * Get attributes for supported primitives.
+ * @param {Element} element - The element to add attributes to.
+ * @returns {PrimitivesAttribute} The primitives attribute object.
+ */
+function getSupportedPrimitivesAttributes(element) {
+	/** @type {PrimitiveTypes} */
+	const primitiveInfo = {
+		supportedPrimitives: []
+	};
+
+	if (isClickable(element)) {
+		primitiveInfo.supportedPrimitives.push('click');
+	}
+	if (isHoverable(element)) {
+		primitiveInfo.supportedPrimitives.push('hover');
+	}
+	if (canInputText(element)) {
+		primitiveInfo.supportedPrimitives.push('input_text');
+	}
+	if (element.tagName.toLowerCase() === 'select') {
+		primitiveInfo.supportedPrimitives.push('select_option');
+	}
+
+	return {
+		'data-supported-primitives': primitiveInfo.supportedPrimitives.join(',')
+	}
+}
+
+/**
+ * Determines if an element is clickable.
+ * 
+ * @param {HTMLElement} element - The DOM element to check.
+ * @returns {boolean} True if the element is considered clickable, false otherwise.
+ */
+function isClickable(element) {
+	if (!element) return false;
+
```

Comment:
Add a `canInputText` check within `isClickable` and return `true` if it is **canInputText**